script.js

```diff
@@ -1,14 +1,22 @@
+const form = document.getElementById('form');
 const container = document.getElementsByClassName('container');
 const searchBar = document.getElementsByClassName('searchbar');
 const submitBtn = document.getElementsByClassName('submitBtn');
-
 const displayContainer = document.getElementsByClassName('displayContainer');
 const temperature = document.getElementsByClassName('temperature');
 const description = document.getElementsByClassName('description');
 const humidity = document.getElementsByClassName('humidity');
 const feelsLike = document.getElementsByClassName('feelsLike');
+const api = '0b8981366e5ef28551fb2db8fa544438';
 
+form.addEventListener("submit", (e) => {
 
-const api = '0b8981366e5ef28551fb2db8fa544438';
+    if (searchBar[0].value.trim() === "") {
```

Comment:
When you done adding [0] above, you can remove it here on this block of code. 

I'm not sure should we use trim() method, because I don't see the logic reason, but we can leave it for now, because function is working.